### PR TITLE
Added MaxRemoteSigns config

### PR DESCRIPTION
**Pull request by TylerS1066**

### CODE REVIEW

**Comment by TylerS1066** [comment]
This should fix #107

**Comment by TylerS1066** [comment]
Note: this isn't tested yet

**Comment by TylerS1066** [comment]
Tested, fixed, retested.  While I can't *confirm* for sure that -1 never errors out, I did launch 625 longarms at once and the remotes all worked (the torpedos, and the server on the other hand....)

**Comment by TylerS1066** [comment]
@cccm5 review please?

File: modules/Movecraft/src/main/java/net/countercraft/movecraft/sign/RemoteSign.java [hunk 1]
```diff
@@ -110,6 +110,13 @@ else if (foundLocations.isEmpty()) {
             event.getPlayer().sendMessage(I18nSupport.getInternationalisedString("ERROR: Could not find target sign!"));
             return;
         }
+        else if (Settings.MaxRemoteSigns != -1) {
```
Comment (cccm5) [inline modules/Movecraft/src/main/java/net/countercraft/movecraft/sign/RemoteSign.java]: else is unnecessary here

Comment (cccm5) [reply modules/Movecraft/src/main/java/net/countercraft/movecraft/sign/RemoteSign.java]: `!=` should be `<=`

**Comment by TylerS1066** [comment]
Fixes implemented

**Comment by cccm5** [comment]
Closes #107

**Pull request merged**